pom.xml

```diff
@@ -162,7 +162,7 @@
             <dependency>
                 <groupId>io.openscore.content</groupId>
                 <artifactId>score-ssh</artifactId>
-                <version>0.0.11</version>
+                <version>0.0.13</version>
```

Comment:
I don't think it's really needed. You told me at one point to test the new SSH from content and make sure it still works with our flows and I committed the change.